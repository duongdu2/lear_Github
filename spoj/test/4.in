8
9
71 53 100 36 46 44 36 26 54 
9
38 31 31 40 97 60 41 50 8 
8
79 49 12 20 88 45 88 17 
5
49 96 35 56 16 
4
82 96 91 57 
7
86 45 63 94 7 94 49 
10
86 24 29 44 81 58 91 17 32 85 
10
46 8 79 96 32 90 94 100 69 59 
