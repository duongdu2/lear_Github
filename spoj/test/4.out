225
207
255
201
269
274
262
290
